new errors with golangci-lint 1.64

```
Linting go files...
WARN The linter 'tenv' is deprecated (since v1.64.0) due to: Duplicate feature another linter. Replaced by usetesting. 
pkg/templates/templating.go:77:7: right hand must be only type assertion (forcetypeassert)
						b, err = raw.(*structpb.Value).MarshalJSON()
						^
pkg/templates/templating.go:85:7: right hand must be only type assertion (forcetypeassert)
						b, err = raw.(*structpb.Value).MarshalJSON()
						^
pkg/templates/templating.go:93:7: right hand must be only type assertion (forcetypeassert)
						s, err := protojson.Marshal(raw.(proto.Message))
						^
pkg/templates/templating.go:101:7: right hand must be only type assertion (forcetypeassert)
						s, err := protojson.Marshal(raw.(proto.Message))
						^
pkg/templates/templating.go:109:7: right hand must be only type assertion (forcetypeassert)
						b, err = json.Marshal(raw.(*structpb.Value).GetBoolValue())
						^
make: *** [Makefile:86: lint-go] Error 1
```

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -115,7 +115,6 @@ linters:
     - stylecheck
     - tagalign
     #- tagliatelle
-    - tenv
     - testableexamples
     # - testifylint
     #- testpackage

--- a/pkg/templates/templating.go
+++ b/pkg/templates/templating.go
@@ -71,44 +71,48 @@ func ReplacePlaceHoldersVariables(template string, dico map[string]string, rawEv
 					if v, ok := val.Value().(string); ok {
 						b = []byte(v)
 					}
-				case types.Bytes:
+
+				case types.Bytes, types.Double, types.Int:
 					raw, err = val.ConvertToNative(structType)
 					if err == nil {
-						b, err = raw.(*structpb.Value).MarshalJSON()
-						if err != nil {
-							b = []byte{}
+						if structVal, ok := raw.(*structpb.Value); ok {
+							b, err = structVal.MarshalJSON()
+							if err != nil {
+								b = []byte{}
+							}
 						}
 					}
-				case types.Double, types.Int:
-					raw, err = val.ConvertToNative(structType)
-					if err == nil {
-						b, err = raw.(*structpb.Value).MarshalJSON()
-						if err != nil {
-							b = []byte{}
-						}
-					}
+
 				case traits.Lister:
 					raw, err = val.ConvertToNative(listType)
 					if err == nil {
-						s, err := protojson.Marshal(raw.(proto.Message))
-						if err == nil {
-							b = s
+						if msg, ok := raw.(proto.Message); ok {
+							b, err = protojson.Marshal(msg)
+							if err != nil {
+								b = []byte{}
+							}
 						}
 					}
+
 				case traits.Mapper:
 					raw, err = val.ConvertToNative(mapType)
 					if err == nil {
-						s, err := protojson.Marshal(raw.(proto.Message))
-						if err == nil {
-							b = s
+						if msg, ok := raw.(proto.Message); ok {
+							b, err = protojson.Marshal(msg)
+							if err != nil {
+								b = []byte{}
+							}
 						}
 					}
+
 				case types.Bool:
 					raw, err = val.ConvertToNative(structType)
 					if err == nil {
-						b, err = json.Marshal(raw.(*structpb.Value).GetBoolValue())
-						if err != nil {
-							b = []byte{}
+						if structVal, ok := raw.(*structpb.Value); ok {
+							b, err = json.Marshal(structVal.GetBoolValue())
+							if err != nil {
+								b = []byte{}
+							}
 						}
 					}
 

